plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-ide/src/main/java/org/eclipse/che/plugin/java/plain/client/wizard/PlainJavaPageViewImpl.java

```diff
@@ -47,14 +45,8 @@
   @Inject
   public PlainJavaPageViewImpl() {
     rootElement = ourUiBinder.createAndBindUi(this);
-
-    browseSourceBtn.addClickHandler(
-        new ClickHandler() {
-          @Override
-          public void onClick(ClickEvent event) {
-            delegate.onBrowseSourceButtonClicked();
-          }
-        });
+    browseSourceBtn.setVisible(false);
```

Comment:
`browseSourceBtn` is always hidden so I guess it's not needed anymore. Should it be removed?

Reply:
Agree but lets do it later )